packages/web-awesome/src/locales/en.json

```diff
@@ -119,7 +122,9 @@
     "attempt": "Attempt {{attempt}} of {{total}}",
     "at": "at",
     "variables": "Variables",
-    "openPwTrace": "Open Playwright Trace"
+    "openPwTrace": "Open Playwright Trace",
+    "finishedAtOriginal": "{{formattedCreatedAt}}, with code {{original}}",
```

Comment:
It may not be obvious to some people that we're talking about the exit codes in a phrase like `August 27, 2025 at 11:10:00 PM, with code 1 (original 1)`. How about keeping the term "exit code" in full? Maybe just `exit code` instead of `with code`?

Reply:
Good idea